dramacool/resolveurl/plugins/dembed.py

```diff
@@ -0,0 +1,99 @@
+"""
+    Plugin for ResolveUrl
+    Copyright (C) 2022 cywteow
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+import re
+import requests
+from resolveurl.lib import helpers
+from resolveurl import common
+from resolveurl.resolver import ResolveUrl, ResolverError
+from resolveurl.plugins.__resolve_generic__ import ResolveGeneric
+from base64 import b64decode, b64encode
+from Cryptodome.Cipher import AES
+from Cryptodome.Util.Padding import unpad
+from Cryptodome.Random import get_random_bytes
+from Cryptodome.Util.Padding import pad
+from base64 import b64decode, b64encode
+from urllib.parse import urlsplit, parse_qs
+from bs4 import BeautifulSoup
+import json
+
+
+class DembedResolver(ResolveGeneric):
+    name = "dembed2"
+    domains = ['dembed2.com']
+    pattern = r'(?://|\.)(dembed2\.com)/(?:streaming)\.php\?id=([a-zA-Z0-9-]+)'
```

Comment:
I think you can also add Asianload too
```suggestion
    pattern = r'(?://|\.)(dembed2\.com)/(?:streaming\.php|embedplus)\?id=([a-zA-Z0-9-]+)'
```